sdks/python/apache_beam/ml/inference/pytorch_inference.py

```diff
@@ -74,6 +74,9 @@ def __init__(
 
       pcoll | RunInference(PytorchModelHandlerTensor(state_dict_path="my_uri"))
 
+    Supported Versions:
+      PyTorch 1.9 and 1.10 are currently supported for Python 3.7, 3.8, and 3.9
+
```

Comment:
Consider moving this below the `Args` section.

Reply:
Done.